tests/test_rds2/test_rds2.py

```diff
@@ -1518,6 +1520,9 @@ def test_create_db_parameter_group():
     db_parameter_group["DBParameterGroup"]["Description"].should.equal(
         "test parameter group"
     )
+    db_parameter_group["DBParameterGroup"]["DBParameterGroupArn"].should.equal(
+        "arn:aws:rds:{0}:REDACTED:pg:{1}".format(region, pg_name)
```

Comment:
The moto.core.ACCOUNT_ID has to be replaced here as well (same in the other test)